src/main/webapp/index.html

```diff
@@ -183,6 +184,18 @@ <h1><a href="/">Census Viz</a></h1>
       <a id="census-link" target="_blank" rel="noopener">Explore more data from the Census API</a>
       <br>
       <div id="data-table"></div>
+      <div id="yearly-total-chart-loading-msg"></div>
```

Comment:
It'd be nice to have an h3 here, like we do for the history and recommendations, saying like "Year-by-year data" or something.

Reply:
Agreed! Added this header